### ISSUE TITLE
Problem: aips are not deleted from the stagging folder when running fixity checks

### ISSUE THREAD

**Issue by scollazo** [archivematica/Issues#1156]
**Expected behaviour**
Once an AIP has been verified by fixity, the temporary folder where it's stored is deleted.

**Current behaviour**
When running fixity against a S3 aipstore, the /var/archivematica/storage_service folder gets filled with temporary folders, until the disk is full.

As a workaround, this line can be addedd to the crontab to delete folders older than 30 minutes: 
```
*/5 * * * *       archivematica   find /var/archivematica/storage_service/ -maxdepth 1 -iname 'tmp*' -mmin +30 -type d -exec rm -rf {} \;
```

**Steps to reproduce**

Scan a fixity aip stored in S3

**Your environment (version of Archivematica, operating system, other relevant details)**
AM qa/1.x , SS qa/0.x in RHEL 7 , using uncompressed AIPs

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged (if applicable)
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by ross-spencer** [comment]
Possible duplicate of https://github.com/archivematica/Issues/issues/431?

**Comment by scollazo** [comment]
Closed as duplicate of #431 , good spooting @ross-spencer !

**Comment by ross-spencer** [comment]
Thanks @scollazo! And I've added your workaround over there now too, it's helpful in the near-term.
